src/modules/consumer.js

```diff
@@ -0,0 +1,95 @@
+var parsers = require('./message-parsers'),
+  utils = require('./utils');
+
+/**
+ * Get a function to execute on incoming messages to handle RPC
+ * @param  {any} msg   An amqp.node message object
+ * @param  {string} queue The initial queue on which the handler received the message
+ * @return {function}       a function to use in a chaining on incoming messages
+ */
+function checkRpc (msg, queue) {
+  /**
+   * When message contains a replyTo property, we try to send the answer back
+   * @param  {any} _content the received message:
+   * @return {any}          object, string, number... the current received message
+   */
+  return (_content) => {
```

Comment:
I didn't pay attention and used old code for this. I will cleanup all occurrences of this